explore/src/main/scala/explore/tabs/ObsTabTiles.scala

```diff
@@ -333,22 +334,28 @@ object ObsTabTiles:
             else angle
 
           val averagePA: Option[AveragePABasis] =
-            (basicConfiguration.map(_.siteFor),
-             props.asterismAsNel,
-             obsDuration.filter(_ > TimeSpan.Zero)
-            )
-              .mapN: (site, asterism, duration) =>
-                props.posAngleConstraint match
-                  case PosAngleConstraint.AverageParallactic =>
-                    val avpa = averageParallacticAngle(
-                      site.place,
-                      asterism.baseTracking,
-                      obsTimeOrNow,
-                      duration
-                    )
-                    flipIfNeeded(avpa)
-                      .map(AveragePABasis(obsTimeOrNow, duration, _))
-                  case _                                     => none
+            (basicConfiguration.map(_.siteFor), props.asterismAsNel, obsDuration, setupTime)
+              .flatMapN: (site, asterism, fullDuration, setupDuration) =>
+                // science duration is the obsDuration - setup time
```

Comment:
and acq time?

Reply:
Yes, the setup time includes the acquisition time.